### PR TITLE
build: enable new merge ready status action

### PR DESCRIPTION
**Pull request by josephperrott**
Enable new action in preperation for sunset of github robot

### CODE REVIEW

File: .github/workflows/merge-ready-status.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# Declare default permissions as read only.
+permissions: {}
+
+jobs:
+  merge-ready-status:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: angular/dev-infra/github-actions/unified-status-check@f1e8862e5034f046cc0bc66641d37f5b36d3e9b5
+        with:
+          angular-robot-key: ${{ secrets.ANGULAR_ROBOT_PRIVATE_KEY }}
```
Comment (devversion) [inline .github/workflows/merge-ready-status.yml:14]: Do you want to set some required statuses, like with the existing robot? or do you plan a follow-up?

File: .github/workflows/merge-ready-status.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: DevInfra
```
Comment (devversion) [inline .github/workflows/merge-ready-status.yml:1]: I still believe this is bad practice tbh. It makes it really hard to filter for the right workflow. See:


image

**Review by devversion — APPROVED** [review]

File: .github/workflows/merge-ready-status.yml (hunk 2 above)
Comment (josephperrott) [reply .github/workflows/merge-ready-status.yml]: Updated

File: .github/workflows/merge-ready-status.yml (hunk 1 above)
Comment (josephperrott) [reply .github/workflows/merge-ready-status.yml:14]: Will do in a follow up

**Review by devversion — APPROVED** [review]

**Pull request closed without merging**